docs/data/routes/docs/nextjs/tracking-and-analytics/configuration/en.md

```diff
@@ -0,0 +1,66 @@
+---
+name: configuration
+routeTemplate: ./data/component-templates/article.yml
+title: Enabling tracking and analytics
+---
+# Walkthrough: Enabling tracking and analytics
+
+This topic will guide you through the steps required to enable full Sitecore tracking and analytics for your Next.js application.
+
+> Note this does not cover client-side tracking via the JSS tracking API, which is possible for *both SSG and SSR*. Please see the [JSS Tracking](/docs/fundamentals/services/tracking) page for details.
+
+## Next.js application
+
+### SSR only
+
+You must use a Next.js SSR page route. An example SSR route is included with the Next.js sample app. See [this page](/docs/nextjs/page-routing/switching-to-ssr) for details.
+
+### Layout Service requests
+
+Sitecore JSS layout service requests must:
+
+1. have tracking enabled (default), and
+2. perform [header passing](/docs/nextjs/tracking-and-analytics/overview#header-passing) (default with the Next.js sample application)
+
+Both of these are taken care of by using the `RestLayoutService` included with the Next.js SDK (part of the `@sitecore-jss/sitecore-jss-nextjs` npm package). 
+
+The `RestLayoutService` will track layout requests by default, so there is no extra configuration required. However, this can be disabled with the optional `tracking` parameter, so ensure this hasn't been specifically set to `false`.
+
+## Sitecore configuration
+
+### Forwarded request header
+
+The `Analytics.ForwardedRequestHttpHeader` Sitecore setting must be set to "X-Forwarded-For".
+
+```xml
+<setting name="Analytics.ForwardedRequestHttpHeader" set:value="X-Forwarded-For" />
+```
+
+> The default Sitecore configuration file included with the Next.js sample app already includes a configuration patch for this setting. You can find this file under `/sitecore/config`.
+
+[Header passing](/docs/nextjs/tracking-and-analytics/overview#header-passing) will send the original IP address of the client on the 'X-Forwarded-For' header. This setting tells Sitecore to read the forwarded header, thus making analytics track the correct original client IP address.
+
+### Disable robot detection
+
+During development, any analytics activity will be flagged as a robot. These settings will enable tracking of robot activity for ease of testing. These should be set in development **only**.
+
+```xml
+<setting name="Analytics.AutoDetectBots" set:value="false" />
+<setting name="Analytics.Robots.IgnoreRobots" set:value="false" />
+```
+
+> The default Sitecore configuration file included with the Next.js sample app already includes a disabled configuration patch for these settings. You can find this file under `/sitecore/config`. Simply uncomment to enable.
+
+## Secure cookies
```

Comment:
I kind of feel like this should be it's own page, since the SSL setup is needed regardless of whether someone is trying to enable analytics or not.

Reply:
It's actually only necessary when you want to start testing analytics. We've got this same bit on other analytics-related "legacy" JSS docs:
https://jss.sitecore.com/docs/fundamentals/services/tracking#jss-tracking
https://jss.sitecore.com/docs/fundamentals/services/layout-service#layout-service-and-analytics